src/main/java/com/epam/ta/reportportal/core/launch/impl/DeleteLaunchHandlerImpl.java

```diff
@@ -121,7 +121,7 @@ public DeleteBulkRS deleteLaunches(DeleteBulkRQ deleteBulkRQ, ReportPortalUser.P
 
 		launchRepository.deleteAll(toDelete);
 		toDelete.stream().map(TO_ACTIVITY_RESOURCE).forEach(it -> {
-			eventPublisher.publishEvent(new DeleteLaunchAttachmentsEvent(it.getId()));
+			attachmentRepository.moveForDeletionByLaunchId(it.getId());
 			messageBus.publishActivity(new LaunchDeletedEvent(it, user.getUserId(), user.getUsername()));
```

Comment:
Do we really need this sent this event?

Reply:
Event doesn't relate to deletion. It is logic of storing reportportal activities.
Method is replaced with more appropriate.

Reply:
It depends of the point of view, i thinks if we implement new logic, it should be clean for us - it's deletions from database + moving data to "deletion table". But if we have some extensible point we may have some unpredictable behavior on it. And for now, when we are trying to simplify this flow and improve performance (it's important, we try to simplify, not to make more extensible and so on). Will be better if we remove all unused events(if it's unused) too.

Reply:
Discussed